### DISCUSSION TITLE
SKIVOX for 121 ( I really need a patch for it lol)

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#938] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
does anyone know if it is really patched?

**Comment by v4pl** [comment]
Yes it is

**Comment by v4pl** [comment]
Don't think you could downgrade anymore.

**Comment by v4pl** [comment, marked as answer]
Just wait for new exploit

**Comment by CGRmags** [comment]
@AshtonDavies help me pls

**Reply by AshtonDavies** [reply]
You cannot downgrade from 121 guys.
Also, the likelyhood of finding a new good exploit is very low now, considering about 8 main exploits are patched according to my record.

@CGRmags SKIOVOX has technically been patched since v119. If the new method does not work for you, it may be patched in that version.

**Reply by CGRmags** [reply]
@AshtonDavies it does work for me my school just hasnet made us upgrade yet and I think I have caub

**Reply by AshtonDavies** [reply]
I really hope Mercury Workshop discovers a new downgrade method for you all. This sucks.

**Reply by CGRmags** [reply]
@AshtonDavies one more thing for the SKIOVOX 119 is the managed window still able to be used if my teacher has hapara?

**Reply by CGRmags** [reply]
LTBEEF flood still works i think?

**Reply by AshtonDavies** [reply]
@CGRmags Your teacher may be able to freeze your screen, because it most likely still runs in the background with the extensions.

Yes, LTMEAT Flood is still not patched.

**Comment by CGRmags** [comment]
ok thanks

On Mon, Feb 19, 2024 at 9:35 AM Ashton Davies ***@***.***>
wrote:

> @CGRmags <https://github.com/CGRmags> Your teacher may be able to freeze
> your screen, because it most likely still runs in the background with the
> extensions.
>
> Yes, LTMEAT Flood is still not patched. I'm gonna push a new update
> somewhat soon to the instructions page that can generate the folder for you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/938#discussioncomment-8520478>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBHYLYY5GJ3PG3ADFA3YUOEN7AVCNFSM6AAAAABDOLXLGKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKMRQGQ3TQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.
